django/core/management/commands/migrate.py

```diff
@@ -134,8 +139,18 @@ def handle(self, *args, **options):
             targets = executor.loader.graph.leaf_nodes()
 
         plan = executor.migration_plan(targets)
-        run_syncdb = options['run_syncdb'] and executor.loader.unmigrated_apps
 
+        if options['plan']:
+            self.stdout.write("Planned operations:", self.style.MIGRATE_LABEL)
```

Comment:
Please use single quotes throughout the patch except if the string contains a single quote.

Reply:
I certainly can do that, though I was trying to emulate the pattern seen below in the file in `master`. 

Lines 156, 187, 192, 195, 309, etc, all use double quotes, even when there are no single quotes.

Please confirm you'd like to shift from that pattern.

Reply:
Yes, for new code we're trying to follow the [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Perfect.